src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/OutOfProcComponent.cs

```diff
@@ -30,12 +30,5 @@ protected void WaitForCompletionSet()
 
         protected void WaitForSignatureHelp()
             => VisualStudioInstance.Workspace.WaitForAsyncOperations(Helper.HangMitigatingTimeout, FeatureAttribute.SignatureHelp);
-
-        protected void WaitForQuickInfo()
-        {
-            VisualStudioInstance.Workspace.WaitForAsyncOperations(Helper.HangMitigatingTimeout, FeatureAttribute.DiagnosticService);
-            VisualStudioInstance.Workspace.WaitForAsyncOperations(Helper.HangMitigatingTimeout, FeatureAttribute.ErrorSquiggles);
```

Comment:
don't we still need to wait on some of these?  otherwise quick info may come up before squiggles have, and then it won't show the diag data in it?  or is that not a problem for some other reason?

Reply:
All calls to `GetQuickInfo` are preceded by calls to `InvokeQuickInfo`. The latter covers the cases.